firmware/README.md

```diff
@@ -433,6 +433,20 @@ Use this console function to force the boron to reset. Useful for development an
 - Nothing if 1 is entered, but does send a message to the particle console to warn about particle's future failed to call message
 - -1 - when bad data is entered
 
+### **reset_stillness_timer_for_alerting_session(String)**
+
+**Description:**
+
+Use this console function to reset the current stillness timer to the current time if and only if the current session has sent at least one alert. This acts to extend the period of time before subsequent Stillness Alerts.
+
+**Argument(s):**
+
+1. Enter any String
+
+**Return(s):**
+
+- The length of the stillness timer before it was reset, converted from a ` long` to an `int`.
```

Comment:
this does return a negative number if the stillness timer runs longer than ~24 days, but I don't think that will be an issue 😆

Reply:
Good point! That is a pretty short time. I don't really have a good concept of how long our Sensors manage to stay online at a time. At least in this case, we don't do anything important with the value on the server side. It's just logged for our information. So if we see a negative number, maybe looking in the docs and seeing that there was a conversion would be enough to give the idea that this might be happening.

Reply:
the state3 timer is reset after a stillness alert is triggered, so it won't come up unless the timeout is >24 days

Reply:
Oh yes, that would be weird. I guess if we had a client who insisted that they only wanted a single stillness alert per session, then we could set the Long Stillness Timer to a really big number and then it might happen. But I think we would discourage that, lol.